### DISCUSSION TITLE
De-Select Once the Batch Edit is Done

### DISCUSSION THREAD

**Discussion by sjugraj** [directus/directus#9791] [Archive]
I don't know is this is a bug or a feature but after a batch Edit is submitted the selection remains checked in the tabular interface.

**Problems**: Sometime if we have a filters applied to remove the rows where some columns is filled. where we do batch edit and provide single value to the column using the multi-select feature and submit the value, but due to the filter the rows are now hidden and user can forget to reset selection.

This can cause the selection to keep on growing in the background and user keep providing the batch-edit and we can lose the data.

**Comment by azrikahar** [comment]
Thanks for reporting this 👍 This does feel like it'll cause many unintended changes, so I've made a PR #9798 to clear selections after saving.

**Reply by sjugraj** [reply]
I have raised an issue related tot he changes i saw in the PR https://github.com/directus/directus/pull/9798#issuecomment-968561942
